expressEngine/order/service/order.js

```diff
@@ -2,18 +2,18 @@
 import { BaseService, DBService } from '../../lib/service/index';
 
 // import collections
-import { Order } from '../../model/index';
+import { order } from '../../model/index';
 
 // import messages
 import { success } from '../../cms/order';
 
 class Service extends BaseService {
   generateOrder = async (data) => {
     try {
-      const { orderNumber, supplierId, unitPrice, productPackage } = data,
-      requiredFields = ['orderNumber', 'supplierId', 'unitPrice', ]
-      this.validateRequired(data, requiredFields)
```

Comment:
Why are you not checking the required fields? Can you please explain me?

Reply:
Because this is service file and we will be validating and using the required field in the parser.